lib/index.coffee

```diff
@@ -6,13 +6,13 @@ path = require 'path'
 # we need a blacklist rather than just a whitelist. this issue is caused by the
 # fact that roots doesn't re-run detection when a file's extension changes.
 BLACKLISTED_FILE_EXTENSIONS = [
-  '.jpg', '.gif', '.png', '.css', '.styl', '.js', '.coffee'
+  '.jpg', '.gif', '.png', '.css', '.styl', '.js', '.coffee', '.scss'
 ]
 
 module.exports = (data) ->
   class TumblrCompiler
     constructor: ->
-      @data = {}
+      @data = data || require './default'
```

Comment:
actually, if that function wrapper isn't needed, then it should be kept in the constructor and we should just get rid of the wrapper. then we could do:

```coffee
class TumblrCompiler
  constructor: (@data) ->
    @data ?= require './default'
```